colossalai/fx/passes/algorithms/ckpt_solver_pofo.py

```diff
@@ -507,6 +507,10 @@ def solver_pofo(gm: ColoGraphModule,
     mem_limit -= parameter_size(gm)
 
     # prepare data
+    from colossalai import META_COMPATIBILITY
+    if META_COMPATIBILITY:
+        from colossalai.fx.profiler import MetaTensor
```

Comment:
Import it on the top of the file lol

Reply:
This is for compatibility.

Reply:
torch 11 will do `MetaInfoProp` with a normal tensor